package.json

```diff
@@ -51,7 +51,9 @@
     "strong-statsd": "^2.0.0",
     "strong-trace": "^1.2.0",
     "strong-url-defaults": "^1.2.0",
-    "strongloop-license": "^1.3.0"
+    "strongloop-license": "^1.3.0",
+    "appmetrics":"RuntimeTools/appmetrics#adapter"
```

Comment:
sort the dependencies (npm will do it for you if you do `npm install --save RuntimeTools/appmetrics#adapter` - with a recent enough npm.